package.json

```diff
@@ -1100,6 +1161,67 @@
                                 "description": "The path to the package.json for the application."
                             }
                         }
+                    },
+                    "python": {
+                        "oneOf": [
```

Comment:
There may be a clever way of using `oneOf` and `required` to eliminate the repetition in these properties.